internal/provider/asset/aws/ecs_compute/models.go

```diff
@@ -144,6 +141,14 @@ func planToAssetInput(ctx context.Context, plan ResourceModel) (cac.AssetInput,
 		},
 	}
 
+	if len(plan.ConnectsTo) > 0 {
```

Comment:
sort order is important

Reply:
we might be able to get away with it but we should worry about, but always sort this alphanumeric (lowest to highest)

Reply:
User provides sort order, I'm not sure it's something I can easily change because if the plan doesn't match state then it'll complain there's a difference.

I can see if that's true for arrays -- I'm assuming it is.